examples/automm/utils.py

```diff
@@ -0,0 +1,237 @@
+'''
```

Comment:
I think it makes sense to move the download utils to https://github.com/awslabs/autogluon/blob/master/text/src/autogluon/text/automm/utils.py